### PR TITLE
DTSPO-264 DB Reader access

### PR DESCRIPTION
**Pull request by timja**
### JIRA link ###
https://tools.hmcts.net/jira/browse/DTSPO-264

### Change description ###
Adds an AAD group which is granted read access

All devs will be added to the non production one, and the production one will be granted through just in time access

Draft: All AAD groups need to be created ahead of time so can't merge till that's done

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by adusumillipraveen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,75 +50,76 @@ The following parameters are optional
...
+GitHub repo.
+
+Typically, you will need to set up an SSH tunnel to access the database you want to.
 
-The password can be retrieved from vault if you've stored it there, or if you view your applications settings.
+All developers can access non production databases with reader access.
 
-Currently developers can only access databases in the sandbox subscription
-The DevOps team can access nonprod databases from the preprod backup box and production databases from the production backup box
+Security cleared developers can access production DBs using just in time access and an approved business justification.
```
Comment (naikajah) [inline README.md:73]: based on below `DTS JIT Access ${var.product} DB Reader SC` 

Is my understanding correct that developer will only be able to request reader access to production database?

File: README.md [hunk 2]
```diff
@@ -30,75 +50,76 @@ The following parameters are optional
...
 
-We're currently working on rolling out access via Azure Active Directory, if you've been granted access you can authenticate with:
+_Note: access is only granted on a case by case basis, and is removed automatically_
```
Comment (naikajah) [inline README.md:75]: It is mentioned about automatic removal. What about the approval process? What will happen once the developer have requested JIT production DB access? Is it approved automatically?

File: README.md (hunk 1 above)
Comment (timja) [reply README.md:73]: There will be follow up work for write, read is the priority as 95% of requests are read

File: README.md (hunk 2 above)
Comment (timja) [reply README.md:75]: No it’s mentioned in here more details to come, that’s being worked out currently.

The proposal is currently the DB owner and someone from senior management have to approve. I’ll probably document that on hmcts.GitHub.io and leave this for the technical bit of connecting

File: set-postgres-permissions.bash [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+CI_AZURE_FOLDER="${HOME}/.azure-${AZURE_SUBSCRIPTION_SHORT_NAME}"
+
+if [ -d "${CI_AZURE_FOLDER}" ]
+then
+    echo "Overriding AZURE_CONFIG_DIR to ${CI_AZURE_FOLDER}"
+    export AZURE_CONFIG_DIR="${CI_AZURE_FOLDER}"
+else
+    echo "Directory ${HOME}/.azure-${AZURE_SUBSCRIPTION_SHORT_NAME} does not exist."
```
Comment (luigibk) [inline set-postgres-permissions.bash:10]: Is that all? No `Warning:`, no `Error:`?

Comment (timja) [reply set-postgres-permissions.bash:10]: no this is designed to run on azure devops too which doesn't do that weird folder thing

**Pull request merged**